app/mesh/access-audit.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Access Audit"
+description: "Track all user and system actions in {{site.mesh_product_name}} using the AccessAudit resource and configurable backends"
+content_type: reference
+layout: reference
+products:
+    - gateway
+
+tags:
+  - regions
+  - geos
+  - network
+
+works_on:
+  - on-prem
+
+related_resources:
+  - text: "Mesh"
+    url: /mesh/overview/
+---
+Access Audit allows you to track all actions executed in {{site.mesh_product_name}}, including actions performed by users and by the Control Plane.
+
+## AccessAudit Resource
+
+The `AccessAudit` resource defines which actions and resource types should be audited. It is global-scoped, meaning it applies across all meshes.
+
+{% navtabs "resource"%}
+{% navtab "Universal" %}
+```yaml
+type: AccessAudit
+name: audit-1
+rules:
+- types: ["TrafficPermission", "TrafficRoute", "Mesh"] # list of types which should be audited. If empty, then default types are audited (see "Default types" below).
+  mesh: default # Mesh within which access to resources is granted. It can only be used with the Mesh-scoped resources and Mesh itself. If empty, resources from all meshes will be audited.
+  access: ["CREATE", "UPDATE", "DELETE"] # an action that is bound to a type.
+  accessAll: true # an equivalent of specifying all possible accesses. Either access or access all can be specified.
```

Comment:
```suggestion
- types: ["TrafficPermission", "TrafficRoute", "Mesh"] 
  mesh: default 
  access: ["CREATE", "UPDATE", "DELETE"] 
  accessAll: true 
```
This is hard to read, and it's the same across both codeblocks. Moving it to a table.